flow/envs/multiagent_env.py

```diff
@@ -241,7 +210,7 @@ def reset(self):
                 self.traci_connection.vehicle.remove(veh_id)
                 self.traci_connection.vehicle.unsubscribe(veh_id)
             except (FatalTraCIError, TraCIException):
-                print("Error during start: {}".format(traceback.format_exc()))
```

Comment:
Same here, why do we not want to throw a warning?

Reply:
again, got rid of bug that necessitated this